docs/unxpq-introduction.md

```diff
@@ -7,89 +7,95 @@ displayed_sidebar: '0.3'
 
 ## What is Dagger?
 
-Dagger is an engine that lets you create devops pipelines as regular code running in regular containers, rather than proprietary YAML running on a proprietary CI/CD platform.
+Dagger is a cross-platform CI/CD engine with 3 defining features:
 
-A Dagger pipeline may be used to build, test, deploy, lint, scan, sign, transform, notify... or perhaps all of the above!
+1. Portable. Run your pipelines on any Docker-compatible container runtime. In practice this means all major development and CI environments.
```

Comment:
OCI compatible vs Docker-compatible?

Reply:
Yes I use them as synonyms. I just don't like to use "OCI" because not everyone knows what it is, and it feels like a cryptic acronym.

Reply:
I reworded and don't need "Docker-compatible" anymore